test/inductor/test_cpu_repro.py

```diff
@@ -1730,6 +1730,72 @@ def fn(test_args):
             res_grad = test_args_for_opt["input"].grad
             self.assertEqual(ref_grad, res_grad)
 
+    def test_decomposed_fake_quant_per_channel(self):
+        def fq(input, scales, zero_points, axis, quant_min, quant_max):
+            res = torch.fake_quantize_per_channel_affine(
+                input, scales, zero_points, axis, quant_min, quant_max
+            )
+            return res
+
+        def qdq(input, scales, zero_points, axis, quant_min, quant_max):
+            res = torch.ops.quantized_decomposed.fake_quant_per_channel(
+                input, scales, zero_points, axis, quant_min, quant_max
+            )
+            return res
+
+        def test_eager_aten_fake_quant(
```

Comment:
nit: might be better to rename `test` to be more descriptive, e.g. `run`

Reply:
Thanks for the suggestion and changed the test name.